src/api/ums/umsModule.ts

```diff
@@ -113,13 +135,13 @@ export class UMSModule implements IModule {
    * @param oldPassword {string}
    * @param newPassword {string}
    */
-  public changePassword(alias: string, oldPassword: string, newPassword: string): Promise<IUMSUser> {
+  public changePassword(alias: string, oldPassword: string, newPassword: string): Promise<D> {
```

Comment:
is it part of this feature to make these methods return observables? I guess it would be nice to make it consistent with the other modules.

Reply:
Well, yeah, probably. I will create a separate PR in this feature branch